flume-ng-channels/flume-kafka-channel/src/test/java/org/apache/flume/channel/kafka/TestOffsetsAndMigration.java

```diff
@@ -121,6 +121,7 @@ private void doTestMigrateZookeeperOffsets(boolean hasZookeeperOffsets, boolean
     Context context = prepareDefaultContext(false);
     context.put(ZOOKEEPER_CONNECT_FLUME_KEY, testUtil.getZkUrl());
     context.put(GROUP_ID_FLUME, group);
+    context.put("offsets.topic.replication.factor", "1");
```

Comment:
What does this change bring? Did we need this to get the test working?

Reply:
Removed it now.